### PR TITLE
DCS-1036 alternative times if no availability in amend

### PR DESCRIPTION
**Pull request by louisenorris**

### CODE REVIEW

File: backend/routes/amendBooking/changeVideoLinkController.ts [hunk 1]
```diff
@@ -30,13 +30,43 @@ export default class ChangeVideoLinkController {
...
       const rooms = await this.locationService.getRooms(res.locals, bookingDetails.agencyId)
 
+      const currentBookingDetails = update
+        ? {
+            date: update.date.format(DAY_MONTH_YEAR),
+            courtId: update.courtId,
+            startTimeHours: update.startTime.format(HOURS_TIME),
+            startTimeMinutes: update.startTime.format(MINUTES_TIME),
+            endTimeHours: update.endTime.format(HOURS_TIME),
+            endTimeMinutes: update.endTime.format(MINUTES_TIME),
```
Comment (andrewrlee) [inline backend/routes/amendBooking/changeVideoLinkController.ts:49]: guess this is a bit inconsistent - one version using `Hours` and the other `.format(HOURS_TIME)`

Maybe fix in a follow up?

Comment (andrewrlee) [reply backend/routes/amendBooking/changeVideoLinkController.ts:49]: would be tempted to extract out into a function in `form.ts` like the `toFormValues` in `backend/routes/createBooking/newBooking/form.ts`.

**Review by andrewrlee — APPROVED** [review]
LGTM

File: backend/routes/amendBooking/changeVideoLinkController.ts (hunk 1 above)
Comment (louisenorris) [reply backend/routes/amendBooking/changeVideoLinkController.ts:49]: I think I decided to go with this as the update date and times were moments but the bookingDetails date and times were all strings - could convert update values to strings and use Hours/Minutes methods there as well?

**Pull request merged**